kyaml/filesys/fsondisk.go

```diff
@@ -125,12 +124,15 @@ func (fsOnDisk) ReadDir(name string) ([]string, error) {
 	return result, nil
 }
 
-// ReadFile delegates to ioutil.ReadFile.
-func (fsOnDisk) ReadFile(name string) ([]byte, error) { return ioutil.ReadFile(name) }
+// ReadFile delegates to os.ReadFile.
+func (fsOnDisk) ReadFile(name string) ([]byte, error) {
+	//nolint:wrapcheck
```

Comment:
note: I can't wrap this because kyaml errors does not have proper `Is` support until the dependency bump in https://github.com/kubernetes-sigs/kustomize/pull/4727 merges